<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgEditSection</class>
 <widget class="QDialog" name="DlgEditSection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit wing section</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Coordinates</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel1_2">
          <property name="text">
           <string>Section name</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>leName</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Current coordinates:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbCoordName">
          <property name="text">
           <string>CoordID</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="pbSave">
          <property name="text">
           <string>&amp;Save</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>101</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbChange">
          <property name="text">
           <string>Change...</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbLoadFile">
          <property name="text">
           <string>Load file</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Interpolation</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="rbInterpolate">
        <property name="text">
         <string>Interpolate coordinates</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QRadioButton" name="rbApproximate">
          <property name="text">
           <string>Least-squares fit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>31</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Points</string>
          </property>
          <property name="buddy">
           <cstring>sbNApprox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbNApprox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>12</number>
          </property>
          <property name="value">
           <number>32</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cbReverseParam">
        <property name="text">
         <string>Reverse parametrization</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbMarkAsBreak">
        <property name="text">
         <string>Mark as segment break</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Transformation</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel1">
          <property name="text">
           <string>Origin x</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>sbPosX</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScienceSpinBox" name="sbPosX">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2">
          <property name="text">
           <string>y</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>sbPosY</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScienceSpinBox" name="sbPosY">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel3">
          <property name="text">
           <string>z</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>sbPosZ</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScienceSpinBox" name="sbPosZ">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="textLabel2_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Chord length</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>sbChord</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="textLabel5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>X-Rotation (deg)</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>sbDihedral</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="textLabel5_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Z-Rotation (deg)</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>sbYaw</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="textLabel4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Y-Rotation (deg)</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>sbTwist</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QScienceSpinBox" name="sbChord">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QScienceSpinBox" name="sbTwist">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QScienceSpinBox" name="sbDihedral">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QScienceSpinBox" name="sbYaw">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>211</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbOK">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string>Alt+K</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QScienceSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qsciencespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>pbChange</tabstop>
  <tabstop>pbLoadFile</tabstop>
  <tabstop>rbInterpolate</tabstop>
  <tabstop>rbApproximate</tabstop>
  <tabstop>sbNApprox</tabstop>
  <tabstop>cbReverseParam</tabstop>
  <tabstop>cbMarkAsBreak</tabstop>
  <tabstop>sbPosX</tabstop>
  <tabstop>sbPosY</tabstop>
  <tabstop>sbPosZ</tabstop>
  <tabstop>sbChord</tabstop>
  <tabstop>sbTwist</tabstop>
  <tabstop>sbDihedral</tabstop>
  <tabstop>sbYaw</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>pbOK</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgEditSection</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbOK</sender>
   <signal>clicked()</signal>
   <receiver>DlgEditSection</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbApproximate</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbNApprox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbApproximate</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
